stix_shifter_modules/vectra/stix_translation/query_constructor.py

```diff
@@ -0,0 +1,678 @@
+import json
+import re
+from os import path
+from pyparsing import nestedExpr, White
+from datetime import datetime, timedelta
+from stix_shifter_utils.utils import logger
+from stix_shifter_utils.stix_translation.src.patterns.pattern_objects import ObservationExpression, \
+    ComparisonExpression, ComparisonComparators, Pattern, CombinedComparisonExpression, CombinedObservationExpression
+
+logger = logger.set_logger(__name__)
+
+START_STOP_PATTERN = r"(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(\.\d+)?Z)"
+STOP_TIME = datetime.utcnow()
+DEFAULT_LIMIT = 10000
```

Comment:
what is default limit? why is it `10000`?

Reply:
removed the default limit; it's not required.